### PR TITLE
BO: Easier way to go to Maintenance page while shop is turned off

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | I had some troubles finding maintenance page so I decided to make alert about enabled maintenance mode clickable |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | n/a |
| How to test? | Turn on maintenance mode, go to Maintenance page from link at the top of the page |
| Sponsor company | impSolutions

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: admin-dev/themes/default/template/header.tpl [hunk 1]
```diff
@@ -308,8 +308,9 @@
               {if isset($maintenance_mode) && $maintenance_mode == true}
                 <span class="shop-state" id="maintenance-mode">
                   <i class="material-icons">build</i>
-                  <span class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
-                    title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in maintenance.'}</strong></p><p class='text-left'>{l s='Your visitors and customers cannot access your shop while in maintenance mode.%s To manage the maintenance settings, go to Preferences > Maintenance.' sprintf='<br />'}</p>">{l s='Maintenance mode'}</span>
+                    <a class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/template/header.tpl]: except  if I'm wrong, paragraph element is not allowed in a link

**Comment by mickaelandrieu** [comment]
Definitively a feature I agree with :)

File: admin-dev/themes/default/template/header.tpl (hunk 1 above)
Comment (kpodemski) [inline admin-dev/themes/default/template/header.tpl]: paragraph in span is also not a good idea but this paragraph is only for a tooltip, right?

Comment (mickaelandrieu) [inline admin-dev/themes/default/template/header.tpl]: agree with you paragraph in span is also a bad idea :/ let's validate product interest with @vincentbz before any git update, I dont want you to work for nothing :)

**Comment by vincentbz** [comment]
Hi @kpodemski ,

Thanks for this PR. I just need to test if there is no conflict with another PR made by our team.

**Pull request merged**